distribution/docs/src/main/resources/_contents/extending-contents.adoc

```diff
@@ -817,6 +817,51 @@ It is recommended that:
 * The maven project's pom file should be updated to attach each data/configuration file as an artifact (using the `build-helper-maven-plugin`)
 * Add each data/configuration file to the KAR file by using the `<configfile>` tag in the KAR's `features.xml` file
 
+=== Migration API
+
+${branding} currently has an experimental API for making bundles migratable. Interfaces in `platform/migration/platform-migratable-api` are
+used by the system to identify bundles that provide implementations for import (_coming soon_) and export operations.
+
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+
+When an `migration:export` command is processed, `ConfigurationMigrationManager` will do a look-up for all registered OSGi bundles
```

Comment:
Few things on this paragraph:

1. Typo: `When a`, not `When an`
2. We should mention the Admin Console export as well
3. Don't think we need to mention the `ConfigurationMigrationManager` here
4. Technically, the code will look up the OSGi _services_ implementing the `DataMigratable` or `ConfigurationMigratable` interfaces, not bundles
5. We shouldn't cover the import part of it yet since it's still being worked
6. We should mention that the services that implement one of the migratable interfaces will be called in turn, in any order, and that they don't need to be thread safe
7. We should add that a bundle or feature can have as many services implementing the interfaces as needed